CMakeLists.txt

```diff
@@ -176,6 +176,16 @@ if(CLANG_TIDY)
     add_dependencies(qa clang-tidy)
     function(aktualizr_clang_tidy)
         file(RELATIVE_PATH SUBDIR ${CMAKE_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
+        set(CHECKS "\
```

Comment:
We're currently using `.clang-tidy` to define the list of checks: https://github.com/advancedtelematic/aktualizr/blob/53626c2e2ddae139216fda53523cef7ea687729a/.clang-tidy

We can switch to using CMake if it makes sense but we'll probably want to keep the whole list in one place.